src/5e-SRD-Monsters.json

```diff
@@ -16299,7 +16459,57 @@
     "actions": [
       {
         "name": "Multiattack",
-        "desc": "The gladiator makes three melee attacks or two ranged attacks."
+        "desc": "The gladiator makes three melee attacks or two ranged attacks.",
+        "options": {
```

Comment:
Would nested choices make sense here? I'm honestly not sure but I'm open to ideas for it. Also, this could just be the easiest approach.